config/_base.js

```diff
@@ -100,17 +99,14 @@ config.compiler_vendor = config.compiler_vendor
 // ------------------------------------
 // Utilities
 // ------------------------------------
-config.utils_paths = (() => {
-  const resolve = path.resolve
-
-  const base = (...args) =>
-    resolve.apply(resolve, [config.path_base, ...args])
-
-  return {
-    base   : base,
-    client : base.bind(null, config.dir_client),
-    dist   : base.bind(null, config.dir_dist)
-  }
-})()
+const resolve = path.resolve
+const base = (...args) =>
+  Reflect.apply(resolve, null, [config.path_base, ...args])
```

Comment:
that's a great resource, pretty much says it all... basically, Reflect provides a nice functional global api for certain primitives, ie apply and operators ie. `delete` but it's really a matter of preference, i'm happy to revert this if you prefer legacy apply, delete, etc